README.md

```diff
@@ -57,7 +57,7 @@ These libraries cannot currently be built on Linux releases later than Focal wit
 
   ```bash
   sudo apt update
-  sudo apt install libicu-dev dotnet-sdk-5.0 mono-complete
+  sudo apt install libicu-dev dotnet-sdk-6.0 mono-complete mono-devel msbuild libcanberra-gtk-module
```

Comment:
`mono-complete` depends on `mono-devel`, so it wouldn't be necessary to explicitly install it.

Reply:
@tombogle, I guess remove "mono-complete" from that line?